circle.yml

```diff
@@ -30,12 +30,18 @@ dependencies:
     - google-chrome --version
     - ./bin/circle/upgrade-google-chrome.sh
     - google-chrome --version
+    # Install Yarn
+    - sudo apt-key adv --keyserver pgp.mit.edu --recv D101F7899D41F3C3
+    - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+    - sudo apt-get update -qq
+    - sudo apt-get install -y -qq yarn
```

Comment:
Seems reasonable to have these steps run as part of rake install actually. Means fewer steps to get setup. Also means we dont need to worry about having circle steps get out of sync with our readme instructions.